python/requirements.txt

```diff
@@ -14,3 +14,4 @@ tornado==4.4.2
 sockjs-tornado>=1.0.3
 wsgiref==0.1.2
 Pillow>=3.2.0
+requests>=2.12.4
```

Comment:
Should we switch >= to == ?  we just know that the current version works, we don't know about future versions of the library